paymentsheet/src/main/java/com/stripe/android/paymentsheet/utils/EventReporterProviderUtil.kt

```diff
@@ -18,6 +20,13 @@ internal fun EventReporterProvider(
         LocalCardNumberCompletedEventReporter provides eventReporter::onCardNumberCompleted,
         LocalCardBrandDisallowedReporter provides eventReporter::onDisallowedCardBrandEntered,
         LocalAnalyticsEventReporter provides eventReporter::onAnalyticsEvent,
+        LocalCardScanEventsReporter provides CardScanEventReporterWrapper(
```

Comment:
Now I see. Can we have EventReporter extend the card scan event reporter interface instead?